README.md

```diff
@@ -11,16 +11,13 @@ Go package facilitating writing API applications in a fast and easy manner.
 ### errors
 Definition of common errors.
 
-### logger
-Interface `ServerLogger` implements common logging methods.
-
 ### net
 Common functionality that comes in handy regardless of the used API architecture. `net` currently supports generating request IDs with some helper methods.
 
 ### http
```

Comment:
Before we release a new version, it would be nice to mention that this new minor change is a breaking change and if you want to use an arbitrary 3rd party package, you should stick with `v0.6.2`.

Reply:
@TomasKocman I update the CHANGELOG. I left out a not about sticking to 0.6.2. I'd rather convey the message that we encourage the upgrade to slog. Besides, we won't be supporting the ServerLogger interface anyway.

Reply:
yy, I just added this point as we agreed on it on the agenda when @bafko mentioned it would be good to mention it somewhere that this is breaking change in the API